modules/Book.js

```diff
@@ -0,0 +1,36 @@
+class Book {
+  constructor() {
+    this.listOfBooks = JSON.parse(localStorage.getItem('data')) || [];
+  }
+
+  updateList() {
+    this.listOfBooks = JSON.parse(localStorage.getItem('data'));
+  }
+
+  updateStorage(items) {
+    localStorage.setItem('data', JSON.stringify(items));
+    this.updateList();
+  }
+
+  addBook(title, author) {
+    const allBooks = this.listOfBooks;
+    const updatedData = [
+      ...allBooks,
+      { id: Math.floor(Math.random() * 100) + 1, title, author },
+    ];
+    this.updateStorage(updatedData);
+  }
+
+  removeBook(id) {
+    const allBooks = this.listOfBooks;
+    const updatedData = allBooks.filter((book) => book.id.toString() !== id);
+    this.updateStorage(updatedData);
```

Comment:
- [ ] Great effort working on the awesome book project 👍 , but currently, in your project, I noticed I could not add or remove books, anytime a user tries to add a book using the form in the `Add New` section it's not adding and in the `List` section it's not displaying any added book which makes it difficult to remove a book from the list. Kindly fix this.
- [ ] Also in the `List` section you should not have a form asking us to add a book. the `List` is only meant to show the books we have added from the `Add New` section and also the `List` section gives room to remove any book added. Kindly remove the form.